<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProrateDialog</class>
 <widget class="QDialog" name="ProrateDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>548</width>
    <height>568</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Prorate Checks</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <widget class="QLabel" name="primaryPayee">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>60</y>
     <width>210</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="primaryShare">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>60</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="primaryCharge">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>60</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="textLabel2">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>10</y>
     <width>130</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Prorate By:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="wordWrap">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonDelete">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonNew">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>100</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="Q3Table" name="prorateTable">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>515</width>
     <height>350</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::ClickFocus</enum>
   </property>
   <property name="numRows">
    <number>0</number>
   </property>
   <property name="numCols">
    <number>4</number>
   </property>
   <property name="selectionMode">
    <enum>Q3Table::Single</enum>
   </property>
   <column>
    <property name="text">
     <string>Payee</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Values</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Share</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Charges</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="buttonCancel">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>520</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Cancel</string>
   </property>
   <property name="shortcut">
    <string/>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonOk">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>520</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;OK</string>
   </property>
   <property name="shortcut">
    <string>Alt+O</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="chkShareChgs">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>10</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Share Charges?</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboProrateMode">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>160</width>
     <height>30</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Percentage</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Sex</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Transaction</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Head</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Flat Amount</string>
    </property>
   </item>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Table</class>
   <extends>Q3Frame</extends>
   <header>q3table.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">types/invoice.h</include>
  <include location="local">types/prorate.h</include>
  <include location="local">types/checkfactory.h</include>
  <include location="local">dao/proratedao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
